packages/contracts/contracts/interfaces/IUnirep.sol

```diff
@@ -66,6 +66,7 @@ interface IUnirep {
     error ProofAlreadyUsed(bytes32 nullilier);
     error NullifierAlreadyUsed(uint256 nullilier);
     error AttesterIdNotMatch(uint160 attesterId);
+    error OutOfRange();
```

Comment:
we should also update error message hash in documentation. we forgot to do this in the previous PRs.
I can fix this in another PR.